src/main/java/com/puppycrawl/tools/checkstyle/api/TokenTypes.java

```diff
@@ -1179,6 +1179,19 @@
     /**
      * The {@code void} keyword.
      *
+     * <p>For example:</p>
+     * <pre>
+     * {@code void literal_void(){}}
+     * </pre>
+     * <p>'void' parses as:</p>
+     * <pre>
+     * --METHOD_DEF -&gt; METHOD_DEF
```

Comment:
Remove "--" to indent whole block to left

Reply:
@romani Fixed it.